### PR TITLE
[INS 244 Drew Sukova] Created sets.adoc

### PR DESCRIPTION
**Pull request by lsukova**
Enhanced the sets.adoc page, adding an introduction, implementation, and examples.

### CODE REVIEW

**Comment by dglass19** [comment]
@lsukova Great job! I wanted to get your thoughts on how we could simplify the text under the `Implementation` section. May an example of how it's ok to declare a blank set with `.set()` but how it will error if we attempt to declare it with multiple set items?

File: content/modules/python/pages/sets.adoc [hunk 1]
```diff
@@ -0,0 +1,190 @@
+= Sets
+
+== Introduction
+
+In Python, lists and sets are both mutable. The key difference is that sets are _unordered_ and cannot have duplicate elements in a set. Sets are particularly useful for membership testing and eliminating said duplicates, even if lists are more useful overall.
```
Comment (naomim01) [inline content/modules/python/pages/sets.adoc:5]: mutable= inconsistent, varying, shifting.... can you edit to a different word?

Comment (lsukova) [reply content/modules/python/pages/sets.adoc:5]: Is that necessary? Mutable and immutable are both really common words when referring to the ability to change data structures.

Comment (naomim01) [reply content/modules/python/pages/sets.adoc:5]: do we have this definition cited somewhere or can we refer to it as fixed or adjustable? you are correct that it cant be inconsistent or a variable or shifting but I do think that we either need to have this definition cited  so saying mutable=adjusting or immutable =fixed

Comment (lsukova) [reply content/modules/python/pages/sets.adoc:5]: Yes, there are 4 instances within the Examples Book where mutable is used and defined as a term. The Python Variables page, the Python Tuples page, the Python Lists page, and the first question of the first project for STAT19000 students. Are you wanting me to link to a definition of mutable?

Comment (naomim01) [reply content/modules/python/pages/sets.adoc:5]: that would be awesome!

**Comment by dglass19** [comment]
Preview build looks good. I believe they updated all of the changes that Naomi recommended.

**Pull request merged**